frontend/test/metabase/scenarios/native/native.cy.spec.js

```diff
@@ -648,9 +648,11 @@ describe("scenarios > question > native", () => {
   it.skip("should be able to add new columns after hiding some (metabase#15393)", () => {
     cy.visit("/");
     cy.icon("sql").click();
-    cy.get(".ace_content").type("select 1 as visible, 2 as hidden");
-    cy.get(".NativeQueryEditor .Icon-play")
+    cy.get(".ace_content")
       .as("editor")
```

Comment:
Don't know how this even worked but it did.
Anyway, changes unrelated to this PR. I simply couldn't ignore this and had to fix it asap.